src/coprocessor/dag/rpn_expr/types/expr.rs

```diff
@@ -106,4 +106,17 @@ impl AsRef<[RpnExpressionNode]> for RpnExpression {
     }
 }
 
+impl RpnExpression {
+    /// Gets the field type of the return value.
+    pub fn ret_field_type<'a>(&'a self, schema: &'a [FieldType]) -> &'a FieldType {
```

Comment:
Is this function related to this PR?

Reply:
Nop, but it is also needed for aggregation function framework. Such small changes are too many that it is not very friendly to create a PR for each of them.

Reply:
I do not think so, I think we should always focus on one thing or one group things in one PR. It seems this function is not related to this PR's title and may confuse the reviewers which may not understand the original issue

Reply:
Ok that's fine